docs/docs/http-api.md

```diff
@@ -0,0 +1,69 @@
+# HTTP API reference
+
+Drand provides a JSON HTTP interface that clients can use to fetch randomness from the drand network. If you're using drand in an application, it may be easier and _more secure_ to use one of the [client libraries](/docs/client-libraries/), which will also perform _verification_ of randomness rounds and add additional features like failover, racing, aggregation, and caching.
+
+All that's required is the address of the HTTP interface and way to fetch from HTTP, e.g. `curl`.
+
+The public [League of Entropy](https://blog.cloudflare.com/league-of-entropy/) HTTP APIs are available at:
+
+- [https://drand.cloudflare.com](https://drand.cloudflare.com)
+- [https://drand.protocol.ai:8081](https://drand.protocol.ai:8081)
+- [https://ln.soc1024.com:8888](https://ln.soc1024.com:8888)
+- [https://drand.nikkolasg.xyz:4444](https://drand.nikkolasg.xyz:4444)
+- [https://drand2.kudelskisecurity.com](https://drand2.kudelskisecurity.com)
```

Comment:
@nikkolasg do you have a list of the IPs/domains and ports the LoE members are running their HTTP APIs on? The above ones are the only ones that work.